tools/ci/cibuild.sh

```diff
@@ -381,6 +381,26 @@ function rust {
   fi
 }
 
+function zig {
+  mkdir -p "${prebuilt}"/zig
+  add_path "${prebuilt}"/zig
+
+  if ! type zig &> /dev/null; then
+    case ${os} in
+      Darwin)
+        brew install zig
+        ;;
+      Linux)
+      # Currently Debian don't install zig (only built)
+        export ZIG_HOME=${prebuilt}/zig
+        cd /tmp
+	wget --quiet https://ziglang.org/download/0.9.1/zig-linux-x86_64-0.9.1.tar.xz
```

Comment:
Will the next works?
```
curl -Sl https://ziglang.org/download/0.9.1/zig-linux-x86_64-0.9.1.tar.xz | tar -xC ${prebuilt}/zig/ --strip 1
```

Reply:
I don't know what version of curl you have, but here it reports an error and suggests `-J` for extracting the tar file.

Reply:
I try `curl -Os`, it does not solve?

Reply:
with `curl -SL` you can do all in a single line without need to do `tar && rm && mv && rm -rf`. I did it some time ago and it was working. Maybe this is not the case here.